### ISSUE TITLE
Eventhub/grid/kafka based agent runtime prototype

### ISSUE THREAD

**Issue by jackgerrits** [microsoft/autogen#4261]
I would really like to create an agent runtime that relies on a shared queue that does not require GRPC connections, at least for events. RPC will likely still require some kind of global location.

**Comment by asheeshgarg** [comment]
This is really a great feature to have. Do we have any time line for this?

**Comment by ekzhu** [comment]
@asheeshgarg not at this point. Something would be good for the future. Unless the community picks up now.

**Comment by mennolaan** [comment]
Yes! A serverless agent that can subscribe to events but also is able to send out events would be very useful. The only way to tackle it now is creating a serverless api that serves as an intermediate that has an api endpoint and sends out events. You could also subscribe this api to all topics so you can send out parallel api calls. But its just a bit less pretty this way.
